Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine/git
+EXPOSE 1737
+
+WORKDIR /
+RUN mkdir wiki
+RUN wget "https://github.com/bouncepaw/mycorrhiza/releases/download/v1.1.0/mycorrhiza-v1.1.0-linux-386.tar.gz" -O mycorrhiza.tar.gz
```

Comment:
Wouldn't it be better to run `docker build` in the source code folder? (cloned from GitHub, for example). This link has to be changed for every new release, not really convenient :/